worlds/cv64/client.py

```diff
@@ -0,0 +1,168 @@
+            # If we are in any other state, such as the Game Over state, set currently_dead to false, so we can once
+            # again send a DeathLink once we are back in the Gameplay state.
+            if game_state not in [0x00000002 or 0x0000000B]:
+                self.self_induced_death = False
+                return
+
+            # Enable DeathLink if it's on in our slot_data
+            if ctx.slot_data is not None:
+                if ctx.slot_data["death_link"]:
+                    if not self.death_link:
+                        self.death_link = True
+                        await ctx.update_death_link(self.death_link)
+
+            # Send a DeathLink if we died on our own independently of receiving another one.
+            if "DeathLink" in ctx.tags and save_struct[0xA4] & 0x80 and not self.self_induced_death and not \
+                    deathlink_induced_death:
+                self.self_induced_death = True
+                await ctx.send_death("Dracula wins!")
+
+            # Increase the game's number of received DeathLinks if the client has received some. To minimize Bizhawk
+            # Write jank, the DeathLink write will be prioritized over the item received one.
+            if self.received_deathlinks and not self.self_induced_death:
+                written_deathlinks += self.received_deathlinks
+                self.received_deathlinks = 0
+                await bizhawk.write(ctx.bizhawk_ctx,
+                                    [(0x389BE2, [written_deathlinks >> 8, written_deathlinks], "RDRAM")])
+            else:
+                # If the game hasn't received all items yet, the received item struct doesn't contain an item, the
+                # current number of received items still matches what we read before, and there are no open text boxes,
+                # then fill it with the next item and write the "item from player" text in its buffer. The game will
+                # increment the number of received items on its own.
+                if num_received_items < len(ctx.items_received):
+                    next_item = ctx.items_received[num_received_items]
+                    if next_item.flags & 0b001:
+                        text_color = [0xA2, 0x0C]
+                    elif next_item.flags & 0b010:
+                        text_color = [0xA2, 0x0A]
+                    elif next_item.flags & 0b100:
+                        text_color = [0xA2, 0x0B]
+                    else:
+                        text_color = [0xA2, 0x02]
+                    received_text, num_lines = cv64_text_wrap(f"{ctx.item_names[next_item.item]}\n"
+                                                              f"from {ctx.player_names[next_item.player]}", 96)
+                    await bizhawk.guarded_write(ctx.bizhawk_ctx,
+                                                [(0x389BE1, (next_item.item & 0xFF).to_bytes(1, "big"), "RDRAM"),
```

Comment:
Not really harmful, but `(next_item.item & 0xFF).to_bytes(1, "big")` can just be `[next_item.item & 0xFF]`.